phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataProtocol.java

```diff
@@ -145,6 +146,8 @@
     }
     
     public static final String CURRENT_CLIENT_VERSION = PHOENIX_MAJOR_VERSION + "." + PHOENIX_MINOR_VERSION + "." + PHOENIX_PATCH_NUMBER;
+    public static final List<String> COMPATIBLE_CLIENT_VERSIONS = 
```

Comment:
Is there no way to get this info programatically? We will have to keep updating this version with each release right? If not, let's document it in the release webpage.

Reply:
There is no easy way to get this programatically as the number of backward compatible versions can depend on the current versions. Sometimes the first code cut of major version might not have any backward compatible clients as per the doc. I have filed a JIRA to add it to the release web page : https://issues.apache.org/jira/browse/PHOENIX-5746